demisto_sdk/commands/download/downloader.py

```diff
@@ -446,10 +446,19 @@ def get_system_automation(self, req_type):
             automation_list.append(ast.literal_eval(api_response[0]))
         return automation_list
 
-    def arrange_response(self, system_items_list):
-        if self.system_item_type == "Playbook":
-            system_items_list = system_items_list.get("playbooks")
+    def get_system_playbook(self, req_type):
+        playbook_list: list = []
+        for playbook in self.input_files:
+            endpoint = f"/playbook/{playbook}/yaml"
+            api_response = demisto_client.generic_request_func(
+                self.client, endpoint, req_type
+            )
+            playbook_received = ast.literal_eval(api_response[0])
```

Comment:
Please avoid `ast.literal_eval`, see the upload rewrite PR (or ask Ilan) for how it's done